contribute/contribute_documentation.rst

```diff
@@ -70,19 +124,78 @@ Checkout a new/separate branch for your issue. For example:
 
         [you@your_laptop rockstor-doc]# git checkout -b issue#1234_brief_label
 
-You can then start making changes in this branch. Once done you can add your
-changes.
+You can then start making changes in this branch.
+
+
+Guidelines
+^^^^^^^^^^
+
+To keep in line with Rockstor's goal to make its features as accessible as
+possible, this documentation should strive to keep non-technical users as its
+primary target. As such, the use of external references and links to additional
+documentation to provide the reader with further technical information is
+encouraged.
+
+If you are generating videos, and prefer uploading your videos to the Rockstor
+Youtube channel, please send an email to support@rockstor.com.
```

Comment:
```suggestion
```
Given we have recently lost the prior main Rockstor YouTube channel of Rockstorinc and have only the barely visible/populated "Rockstor" channel left, and not under my direct control (we have to bother Suman for that) I'm thinking of cutting our loses and keeping things as simple as possible and removing, in time, all YouTube links. I'm not myself keen on the video links and they are a lot of work to maintain. Once we have openQA testing in place we may have a source to reproduce current functions and even, potentially, video generation on-hand. But we are not there yet unfortunately. Hence suggesting we scrub this suggestion as we already have a backlog of now 'lost' videos that we either have to re-create and host or work around them.